Morseovka.py

```diff
@@ -0,0 +1,90 @@
+#Definování morseovky
+MORSE_CODE_DICT = {
+    'A':'.-',
+    'B':'-...',
+    'C':'-.-.',
+    'D':'-..',
+    'E':'.',
+    'F':'..-.',
+    'G':'--.',
+    'H':'....',
+    'I':'..',
+    'J':'.---',
+    'K':'-.-',
+    'L':'.-..',
+    'M':'--',
+    'N':'-.',
+    'O':'---',
+    'P':'.--.',
+    'Q':'--.-',
+    'R':'.-.',
+    'S':'...',
+    'T':'-',
+    'U':'..-',
+    'V':'...-',
+    'W':'.--',
+    'X':'-..-',
+    'Y':'-.--',
+    'Z':'--..',
+    '1':'.----',
+    '2':'..---',
+    '3':'...--',
+    '4':'....-',
+    '5':'.....',
+    '6':'-....',
+    '7':'--...',
+    '8':'---..',
+    '9':'----.',
+    '0':'-----',
+}
+
+def zakodovani(text): #funkce pro zakodovani
+    zakodovany_text = "" #zakodovany text = string
+    for pismena in text: 
+        if pismena != " ": #kontrola mista
+            zakodovany_text = zakodovany_text + MORSE_CODE_DICT.get(pismena) + " " #vyhleda slovnik a prida odpovidajici znak + mezeru
```

Comment:
tady to generuje problém s mezerou přidávanou navíc na konci překladu (za posledním zakódovaným písmenem)